index.js

```diff
@@ -1,23 +1,23 @@
 var plugins = {
-	usedKeywords: require( "./js/bundledPlugins/previouslyUsedKeywords" )
+	usedKeywords: require( "./build/bundledPlugins/previouslyUsedKeywords" ),
 };
 
 var helpers = {
-	scoreToRating: require( "./js/interpreters/scoreToRating" )
+	scoreToRating: require( "./build/interpreters/scoreToRating" ),
 };
 
 module.exports = {
```

Comment:
This will require people that use this library and do not import everything from index.js, to change every import from `./js/...` to `./build/...`.